src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -292,18 +293,19 @@ public String toString() {
 
 enum SQLServerDriverBooleanProperty
 {
-	DISABLE_STATEMENT_POOLING          ("disableStatementPooling",         true),
-	ENCRYPT                            ("encrypt",                         false),	
-	INTEGRATED_SECURITY                ("integratedSecurity",              false),
-	LAST_UPDATE_COUNT                  ("lastUpdateCount",                 true),
-	MULTI_SUBNET_FAILOVER              ("multiSubnetFailover",             false),
-	SERVER_NAME_AS_ACE                 ("serverNameAsACE",                 false),
-	SEND_STRING_PARAMETERS_AS_UNICODE  ("sendStringParametersAsUnicode",   true),
-	SEND_TIME_AS_DATETIME              ("sendTimeAsDatetime",              true),
-	TRANSPARENT_NETWORK_IP_RESOLUTION  ("TransparentNetworkIPResolution",  true),
-	TRUST_SERVER_CERTIFICATE           ("trustServerCertificate",          false),
-	XOPEN_STATES                       ("xopenStates",                     false),
-	FIPS                               ("fips",                            false);
+	DISABLE_STATEMENT_POOLING                 ("disableStatementPooling",                   true),
+	ENCRYPT                                   ("encrypt",                                   false),	
+	INTEGRATED_SECURITY                       ("integratedSecurity",                        false),
+	LAST_UPDATE_COUNT                         ("lastUpdateCount",                           true),
+	MULTI_SUBNET_FAILOVER                     ("multiSubnetFailover",                       false),
+	SERVER_NAME_AS_ACE                        ("serverNameAsACE",                           false),
+	SEND_STRING_PARAMETERS_AS_UNICODE         ("sendStringParametersAsUnicode",             true),
+	SEND_TIME_AS_DATETIME                     ("sendTimeAsDatetime",                        true),
+	TRANSPARENT_NETWORK_IP_RESOLUTION         ("TransparentNetworkIPResolution",            true),
+	TRUST_SERVER_CERTIFICATE                  ("trustServerCertificate",                    false),
+	XOPEN_STATES                              ("xopenStates",                               false),
+    FIPS                                      ("fips",                                      false),
```

Comment:
Nit: Indentation is off compared to other items. 
Also why did you change the formatting of the rest of the enums? 
Having only the lines which really changed, makes code reviews easier, and prevents unnecessary merge conflicts if other devs are working on the same region of code.